src/Message/PurchaseRequest.php

```diff
@@ -62,13 +62,41 @@ protected function getFunction()
         return 'authCapture';
     }
 
-    public function getData($paymentMethodType = self::PAYMENT_METHOD_CREDIT_CARD)
+    public function getData(string $paymentMethodType = self::PAYMENT_METHOD_CREDIT_CARD)
```

Comment:
This repo supports PHP 5.3+ and scalar type hinting is a PH 7+ feature so this repo needs to rely on PHPdocs

Reply:
bummer. :( thanks, I'll change that